<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsItemWidget</class>
 <widget class="QWidget" name="SettingsItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SettingsItemWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">labelTitle:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineItem"/>
     </item>
     <item>
      <widget class="KEditListBox" name="listItems"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboItems"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
